src/ShopPaymentTransactionBundle/objects/TPkgShopPaymentTransactionManagerEndPoint.class.php

```diff
@@ -319,7 +377,8 @@ public function hasTransactions($sTransactionType = null)
         }
         $query = "SELECT COUNT(*) AS transactions
                     FROM `pkg_shop_payment_transaction`
-                   WHERE `pkg_shop_payment_transaction`.`shop_order_id` = '".MySqlLegacySupport::getInstance()->real_escape_string(
+                   WHERE `pkg_shop_payment_transaction`.`shop_order_id` = '".MySqlLegacySupport::getInstance(
```

Comment:
There's a lot of formatting in this PR. This is somewhat OK, but it looks like you're not formatting code by using PHP-CS-Fixer with the given config. Please use this config in general by navigating to the bundle root, installing dependencies through composer and then calling php-cs-fixer (installed e.g. globally on your system) without arguments.